pvlib/bifacial/infinite_sheds.py

```diff
@@ -283,8 +286,7 @@ def _vf_row_ground_integ(f_x, surface_tilt, gcr, npoints=100):
     given by
 
     .. math ::
-        \\large{f_{gr} = \frac{1}{2} \\left(\\cos\\left(\\psi_t\\right) -
-        \\cos \\left(\\beta\\right) \\right)
+        \\large{f_{gr} = \frac{1 - \\cos\\left(\\beta - \\psi_t\\right)}{2}
```

Comment:
```suggestion
        \\large{f_{gr} = \frac{1 - \\cos\\left(\\psi_t - \\beta\\right)}{2}
```

Note: the order it doesn't matter because by [trig identity](https://en.wikipedia.org/wiki/List_of_trigonometric_identities#Reflections):
$$\cos\left(\theta\right) = \cos\left(-\theta\right)$$
but this order matches the order in `_vf_row_ground()` above and also the order in `_vf_row_sky_integ()` and in its LaTeX equation.

BTW: now GitHub renders LaTeX math? wow!